extensions/extension.go

```diff
@@ -0,0 +1,34 @@
+package extensions
+
+import (
+	"math"
+
+	"grol.io/grol/object"
+)
+
+func init() {
+	Init()
+}
+
+func Init() {
+	cmd := object.Extension{
+		Name:     "pow",
+		MinArgs:  2,
+		MaxArgs:  2,
+		ArgTypes: []object.Type{object.FLOAT, object.FLOAT},
+		Callback: pow,
+	}
+	err := object.CreateCommand(cmd)
+	if err != nil {
+		panic(err)
+	}
+}
+
+func pow(args []object.Object) object.Object {
```

Comment:
Can't use a variadic or simply have 2 parameter, it might be a huge refactoring but it would be better I think

Reply:
what do you mean? MaxArgs -1 is variadic (and variadic is same as [] anyway)

Reply:
It could be pow(val, exp)

Or pow(args ...object.Object)

Reply:
yes... so that would be like it is today MinArgs=MaxArgs=2

the second one is MinArgs=0 MaxArgs=-1

Reply:
See https://github.com/grol-io/grol/pull/96/commits/7d759b8137dc5947bb372e3de1dd079ba5de80e3 for `sprintf` integration as variadic example